src/docs/product/sentry-basics/guides/alert-notifications/issue-alerts.mdx

```diff
@@ -19,7 +19,7 @@ Each environment typically necessitates different levels of urgency and workflow
 
 Manage your alert rule by giving it a descriptive name. A descriptive alert rule name specifies the team affected and the topic of the alert. For example, "Frontend Latency," "Backend Failure Rate," or "Billing Apdex."
 
-You can also set the team that should be notified of this alert. Only teams that you're a member of are available for you to select. The team that you set here owns the alert and is automatically assigned to it.
+You can choose a team to associate with the alert. When you do so, members of the selected team can edit this alert. Only teams that you're a member of are available for you to choose.
```

Comment:
```suggestion
You can choose a team to associate with an alert so that members of the that team can edit this alert. Note that you can only make this association if you are a member of the team.
```